src/app/demos/layouts/dynamic-sidebar/dynamic-sidebar-demo.component.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  StacheNavLink
+} from '../../../public';
+
+@Component({
+  selector: 'app-dynamic-sidebar-demo',
+  templateUrl: './dynamic-sidebar-demo.component.html'
+})
+export class DynamicSidebarDemoComponent {
+
+  // NOTE: To avoid the sidebar "flashing" the default navlinks,
+  // you must provide an array with at least one item.
```

Comment:
For more info, the following line ignores empty arrays: https://github.com/blackbaud/skyux-lib-stache/blob/master/src/app/public/modules/sidebar/sidebar.component.ts#L31